chocolatey/tests/integration/targets/win_chocolatey_facts/tasks/test_gather_facts.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Ensure there are no facts from previous tests
+  ansible.builtin.meta: clear_facts
+
+- name: Gather all facts, with implied "all" filter
+  win_chocolatey_facts:
+
+- name: assert 'chocolatey outdated' facts
```

Comment:
There's three tests in this file with this same name that are all asserting different things, so we should probably name them appropriately. Also, none of them are specifically for picking just outdated facts; should we have one for that? 😁

Reply:
🤦‍♂️ That was a little silly of me, I must have been blind to the asserts. Fixed up and squashed!